client/packages/core/src/index.ts

```diff
@@ -489,11 +490,11 @@ class InstantCoreDatabase<Schema extends InstantSchemaDef<any, any, any>>
    *    console.log(resp.data.goals)
    *  });
    */
-  subscribeQuery<Q extends InstaQLParams<Schema>>(
-    query: Q,
-    cb: (resp: InstaQLSubscriptionState<Schema, Q>) => void,
-  ) {
-    return this._reactor.subscribeQuery(query, cb);
+  subscribeQuery<Q extends InstaQLParams<Schema>>(query: Q, cb: (resp: InstaQLSubscriptionState<Schema, Q>) => void);
```

Comment:
Nit: I would suggest putting opts as the third arg, mainly to avoid overloads. 

Afaik overloads don't show up well in intellisense. For example, if you hover over the function, it will now choose the first overload, and then say (+ 1 overload), without explaining the other overloads. 

At least as per this discussion: https://stackoverflow.com/questions/79187112/what-is-the-idiomatic-way-to-make-a-generic-type-optional#comment139640779_79189035

Reply:
Yeah. My concern was that callback works best when it’s the last argument. But I agree it’s better when arguments don’t change their position in overloads